opty/direct_collocation.py

```diff
@@ -166,6 +172,13 @@ def __init__(self, equations_of_motion, state_symbols,
             ndarrays of floats of shape(N,). Any time varying parameters in
             the equations of motion not provided in this dictionary will
             become free trajectories optimization variables.
+        known_experiment_map : dictionary, optional
+            A dictionary that maps the state SymPy functions of time to
+            ndarrays of floats of shape(N,4). Any time varying parameters in
+            the equations of motion not provided in this dictionary will
+            become free trajectories optimization variables.
+	    Lamda: float
+	        A parameter in homotopy method that controls the change of motion equations.
```

Comment:
We should be careful with the name of this. `lambda` is a reserved word in python, it is a built in function. And sympy has `Lambda` which is a symbolic version of the Python `lambda` and creates a symbolic function. Maybe a variable name like `homotopy_control` or something similar is better.